### ISSUE TITLE
Add command for renaming a page

### ISSUE THREAD

**Issue by richtabor** [WordPress/gutenberg#60089]
It's difficult to change the name of a page from the Site Editor, if the page template does not have a title block, as originally detailed in https://github.com/WordPress/gutenberg/issues/52763. While https://github.com/WordPress/gutenberg/pull/54648 adds a control to the Inspector, it'd be nice to have a rename command (as we do for patterns). 

![CleanShot 2024-03-21 at 13 56 50](https://github.com/WordPress/gutenberg/assets/1813435/aa30ad5e-caef-4276-b621-5cc50b19f392)

![CleanShot 2024-03-21 at 14 05 34](https://github.com/WordPress/gutenberg/assets/1813435/aa2a48ab-9e7a-47bd-8329-906f1e752434)

**Comment by aaronrobertshaw** [comment]
There's also some ongoing work adding a commands to [rename](https://github.com/WordPress/gutenberg/pull/55339) and [duplicate](https://github.com/WordPress/gutenberg/pull/55342) template parts. As part of that I'd be happy to link up with @SaxonF and progress https://github.com/WordPress/gutenberg/pull/54648 and this along.

**Comment by SaxonF** [comment]
@aaronrobertshaw yes please! That PR is quite old so might be a challenge bringing it up to date .

**Comment by aaronrobertshaw** [comment]
A PR adding a rename page command is available for early feedback and review in https://github.com/WordPress/gutenberg/pull/60231.

**Comment by t-hamano** [comment]
It has been suggested to add a command for renaming outside of the site editor as well: #70969

**Comment by talldan** [comment]
I've closed https://github.com/WordPress/gutenberg/issues/70969 as a duplicate of this. Ideally I think there'd be one command that works universally.
